package.json

```diff
@@ -19,14 +19,24 @@
     }
   ],
   "devDependencies": {
+    "atob": "^1.1.2",
+    "bluebird": "^2.9.30",
     "chai-spies": "^0.5.1",
     "esperanto": "^0.6.6",
+    "fs-extra": "^0.19.0",
+    "glob": "^5.0.10",
     "grunt": "~0.4.1",
     "grunt-contrib-jshint": "0.11.2",
     "grunt-contrib-uglify": "^0.4.0",
     "grunt-git-authors": "^3.0.0",
     "grunt-jscs": "1.8.0",
-    "intern": "2.2.2"
+    "intern": "2.2.2",
+    "lodash": "^3.9.3",
+    "request": "^2.58.0"
```

Comment:
I know that this is opinionated but it looks like you have to download a half of the internet while getting `request`. Why not to use something more lighter for such a trivial task? How about [got](https://www.npmjs.com/package/got) or [asker](https://www.npmjs.com/package/asker)?